src/index.php

```diff
@@ -1,133 +1,44 @@
 <?php
 
-namespace Islandora\CollectionService;
+namespace Islandora\PDX;
 
 require_once __DIR__.'/../vendor/autoload.php';
 
 use Silex\Application;
-use Islandora\Chullo\Uuid\UuidGenerator;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 use Psr\Http\Message\ResponseInterface;
 use Silex\Provider\TwigServiceProvider;
-use Islandora\ResourceService\Provider\ResourceServiceProvider;
-use Islandora\CollectionService\Provider\CollectionServiceProvider;
-use Islandora\TransactionService\Provider\TransactionServiceProvider;
+use Islandora\PDX\CollectionService\Provider\CollectionServiceProvider
 
 date_default_timezone_set('UTC');
 
 $app = new Application();
 
 $app['debug'] = true;
 $app->register(new \Silex\Provider\ServiceControllerServiceProvider());
-$app->register(
-    new \Silex\Provider\TwigServiceProvider(),
-    array(
-      'twig.path' => __DIR__.'/../templates',
-    )
-);
-
-$islandoraResourceServiceProvider = new ResourceServiceProvider;
-//Registers Resource Service and defines current app's path for config context
-$app->register(
-    $islandoraResourceServiceProvider,
-    array(
-      'islandora.BasePath' => __DIR__,
-    )
-);
-$app->mount("/islandora", $islandoraResourceServiceProvider);
-
-$islandoraTransactionService = new TransactionServiceProvider;
-
-$app->register($islandoraTransactionService);
-$app->mount("/islandora", $islandoraTransactionService);
+// TODO: Not register all template directories right now.
+$app->register(new \Silex\Provider\TwigServiceProvider(), array(
+  'twig.path' => array(
+    __DIR__ . 'CollectionService/templates',
+  ),
+));
 
-$islandoraCollectionService = new CollectionServiceProvider;
+$islandoraCollectionServiceProvider = new CollectionServiceProvider;
 
-$app->register(
-    $islandoraCollectionService,
-    array(
-      'UuidGenerator' => new UuidGenerator(),
-    )
+$basepath = array(
+  'islandora.BasePath' => __DIR__,
 );
```

Comment:
In theory this should work, but instead I would recommend not passing it to the `register()` function and just setting
```
$app['islandora.BasePath'] = __DIR__;
```

Like what I did in [my PR](https://github.com/ruebot/Crayfish/pull/1/files#diff-5472c9619eb85b5fe7fda44283a0f366R21) for Crayfish